tests/contracts/compiled/CallPermit.json

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
This file was probably included by mistake

Reply:
Yes, my bad, I think it was missing when generating the contracts

Reply:
Do we need to remove it?